### ISSUE TITLE
Input field does not follow standard copy-pasting behavior

### ISSUE THREAD

**Issue by kasnerz** [ufal/charles-translator-android#16]
The input field does not support standard Android copy-pasting features (copy / cut / paste / select all) on double-tap or using a keyboard.

I noticed that there is a paste button when the field is empty (and there is also a copy button when there is some text). But I don't think this is the best way to do it, because I often need to paste something after some existing text or copy/cut a text segment.

**Comment by kasnerz** [comment]
I just noticed that long tapping works, but at least to me it seems quite unnatural.

**Comment by krabatom** [comment]
You are right, that there is strange behavior with input. Thank you for report, I will look at it. What keyboard are you using?

**Comment by kasnerz** [comment]
Well, I am using a custom instance of [Mutliling-O keyboard](https://play.google.com/store/apps/details?id=kl.ime.oh&hl=cs&gl=US) which I modelled after the [Messagease](https://play.google.com/store/apps/details?id=com.exideas.mekb&hl=cs&gl=US) keyboard I used previously, but I guess I'm not a typical user :grin: 

![Screenshot_20220304-222400_1x](https://user-images.githubusercontent.com/8361334/165338179-958ff028-3620-4e08-85a5-525ce9e88f59.jpg)

However I believe the behavior of the copy/paste buttons should be fairly standard.
